js/clinicalTimeline.js

```diff
@@ -156,7 +155,7 @@ var clinicalTimeline = (function(){
     d3.select(divId + " svg")
       .insert("rect", ".timeline-label")
       .attr("width", 130)
-      .attr("height", gBoundingBox.height - 15)
+      .attr("height", 245)
```

Comment:
I mean it is used twice in `clinicalTimeline`. By making it a constant it is a little more clear that it is used multiple times. If you prefer not to share it with the other files, you could also add it is a constant only within the scope of `clinicalTimeline`